src/modules/Client/Api/Admin.php

```diff
@@ -567,6 +578,14 @@ public function group_delete($data)
 
         $model = $this->di['db']->getExistingModelById('ClientGroup', $data['id'], 'Group not found');
 
+        $clients = $this->di['db']->find('Client', 'client_group_id = :group_id',[':group_id' => $data['id']]);
+
+        $count = count($clients);
```

Comment:
It's minor, but I'd rather just use the count function in the if statement below.
The other cases are fine as-is as they are doing a lot more than just `count($clients)`, though